src/components/SourcesTree.js

```diff
@@ -156,8 +158,13 @@ class SourcesTree extends Component {
   }
 
   getIcon(item, depth) {
+    let name = supportedLibraries.find(
```

Comment:
we should use a feature flag here: `collapseFrame` so that we can turn this feature on later

```
isEnabled("collapseFrame")
``